ocs_ci/ocs/constants.py

```diff
@@ -1789,7 +1789,7 @@
     "Orange": ["/scale/"],
     "Black": ["/ui/"],
     "Yellow": ["/managed-service/"],
-    "Turquoise": ["/regional-dr/"],
+    "Turquoise": ["/disaster-recovery/"],
```

Comment:
mdr should go to a different squad

Reply:
@am-agrawa which squad mdr belongs to?

Reply:
Harish might know that, hence I tagged him above.

Reply:
@ebenahar @hnallurv correct me here, it was decided both to be in 1 squad for the time being and in case we identify separation is needed which can be done in future that's why the PR raised with same marker #7961

Reply:
@am-agrawa from the @Akarsha-rai update, I didn't see a reason of having CR for this PR. If you feel this PR still needs CR, please provide inputs for the previous comments.

Reply:
Hi @vavuthu I am not sure what's decided and where, so I unblocking the PR.

Reply:
from @Akarsha-rai update

```
@ebenahar @hnallurv correct me here, it was decided both to be in 1 squad for the time being and in case we identify separation is needed which can be done in future that's why the PR raised with same marker https://github.com/red-hat-storage/ocs-ci/pull/7961

```

Reply:
Yes, that's correct.

Reply:
that's correct